src/auto_optional/main.py

```diff
@@ -8,8 +9,10 @@
 
 
 @app.command()
-def main(path: Path = typer.Argument(".")) -> None:
-    converted_files = convert_path(path)
+def main(path: List[Path]) -> None:
```

Comment:
No, unfortunately that doesn't work, because typer throws an error that basically says that default is not allowed, when using multi values.

But there is another solution.
`path: List[Path]=typer.Argument(None)` and setting `path = [Path(".")]` when path is empty.

Reply:
Great, that sounds good.